src/collection/models.rs

```diff
@@ -101,11 +101,33 @@ pub struct Recipe {
     pub body: Option<Template>,
     pub authentication: Option<Authentication>,
     #[serde(default)]
-    pub query: IndexMap<String, Template>,
+    pub query: QueryParameters,
     #[serde(default)]
     pub headers: IndexMap<String, Template>,
 }
 
+#[derive(Clone, Debug, Serialize, Default)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct QueryParameters(Vec<(String, Template)>);
```

Comment:
I don't think this struct is necessary. It seems like the main purpose is to implement custom deserialization, but since it's only used in one spot I think we'd just be better off putting `#[serde(deserialize_with = "deserialize_query_parameters")]` on `Recipe::query`. That will save some boilerplate code.

Reply:
Yep, I was wondering if there is better way to do this